php/commands/core.php

```diff
@@ -99,6 +99,14 @@ public function download( $args, $assoc_args ) {
 			$download_url = str_replace( '.zip', '.tar.gz', $offer['download'] );
 		}
 
+		$from_version = '';
+		$includes_folder = defined( 'WPINC' ) ? WPINC : 'wp-includes';
```

Comment:
Actually, given this command runs on `before_wp_load`, when would it be defined?